assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -137,6 +137,14 @@ che.auth.access_denied_error_page=/error-oauth
 # Reserved user names
 che.auth.reserved_user_names=
 
+### Pull Request Generation
+# If true (default), then generates a Factory URL embedded within the PR
+# generated by Che for provider that manages the pull request. The PR
+# factory is a URL that generates a workspace that contains the commit of
+# the PR itself. Setting this to false will generate a PR without an
+# embedded Factory URL. Generate PRs using the pull request panel in the IDE.
+che.pr=true
```

Comment:
> CHE_PR=true in che.env
> che.pr=true in che.properties
> (PS: changed property name after discussion with Tyler)

I have no idea why the property called `che.pr` is clearer than `che.pr.generate_review_factory`.
First thing i though was that this property enables or disables PR panel appearance.
if we need a short name for it, let's at least make it related to factory e.g. `che.pr.factory=true`